### PR TITLE
Category list

### PR DESCRIPTION
**Pull request by DwinaTech**
added api category list to get category data from db

### CODE REVIEW

File: server/routes/api.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 var express = require('express');
 var router = express.Router();
```
Comment (kabaros) [inline server/routes/api.js:2]: @MrDwina the conflict is not resolved properly (there shouldn't be <<< >>>>

**Review by kabaros — CHANGES_REQUESTED** [review]

File: server/routes/api.js (hunk 1 above)
Comment (DwinaTech) [reply server/routes/api.js:2]: where is this?

File: server/routes/api.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-const articleDB = require('../dbClients/articlesDB')
+const categoryClient = require("../dbClients/categoriesDB");
```
Comment (kabaros) [inline server/routes/api.js]: this is wrong - articlesDB is still needed by /articles. You need both articlesDB and categoriesDB (and all it categoriesDB like the file not categoriesClient)

File: server/routes/api.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 var express = require('express');
 var router = express.Router();
+const categoryClient = require("../dbClients/categoriesDB");
```
Comment (kabaros) [inline server/routes/api.js]: call it categoriesDB (not catergoryClient - normally you match your moduleName)

**Review by kabaros — CHANGES_REQUESTED** [review]

File: server/routes/api.js (hunk 3 above)
Comment (DwinaTech) [reply server/routes/api.js]: alright

**Review by kabaros — APPROVED** [review]

**Pull request merged**